pkgs/servers/misc/navidrome/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "navidrome";
+  version = "0.20.0";
+
+  src = fetchurl {
+    url = "https://github.com/deluan/navidrome/releases/download/v${version}/navidrome_${version}_Linux_x86_64.tar.gz";
+    sha256 = "1k01v0qg7lc30q727s8f1d5p166zx13z9qa7dy33b665768v7ajr";
+  };
+
+  buildCommand = ''
+     tar xvf $src navidrome
```

Comment:
If you use `src = fetchzip {` above, you can drop this line
```suggestion
```

Reply:
I can't do this since I can't use `fetchzip`